doc/api/modules.md

```diff
@@ -914,9 +914,11 @@ added: v10.12.0
   function.
 * Returns: {[`require`][]} Require function
 
+Create a custom require function that will resolve modules relative to the filename path.
+
 ```js
 const { createRequireFromPath } = require('module');
-const requireUtil = createRequireFromPath('../src/utils');
+const requireUtil = createRequireFromPath('../src/utils'); // can also be a file
```

Comment:
Am I mistaken or won't the code fail if `utils` is a directory unless it's like this?:

```suggestion
const requireUtil = createRequireFromPath('../src/utils/.');
```

And if so, then the comment is misleading because it implies that passing a directory without `/.` at the end will work.

And explaining the `/.` requirement is probably more trouble than it's worth.

Probably best to update this to point to a file and not a directory at all.

```suggestion
const requireUtil = createRequireFromPath('../src/utils/index.js');
```

Reply:
Yes we should likely go with this last suggestion please.